src/server/qgsserver.cpp

```diff
@@ -300,9 +300,13 @@ void QgsServer::putenv( const QString &var, const QString &val )
  * @param queryString
  * @return response headers and body
  */
-
 void QgsServer::handleRequest( QgsServerRequest &request, QgsServerResponse &response )
```

Comment:
This method is rededundant and should be removed: since you have defined nullptr as the default value for the overloaded handleRequest method, this signature is duplicated.